docs/source/getting_started/command_line_tool.rst

```diff
@@ -96,14 +96,14 @@ Lightly-magic is a singular command for training, embedding, and uploading to th
     
 .. code-block:: bash
     
-    # embedding and uploading the images and embeddings to web app
-    # passing in input directory of dataset(Data_dir),using pretrained model
-    # with assigned token as yourToken and datasedID as yourDatasedID
-    # loader.num_workers species the number of cpu cores used for speeding the process.
+    # Embedding and uploading the images and embeddings to web app passing in 
+    # input directory of dataset(Data_dir), using pretrained model with assigned 
+    # token as yourToken and datasedID as yourDatasedID loader.num_workers specifies 
```

Comment:
There was a reason why `loader.num_workers` was on a a new line: it is a new sentence.

Reply:
also, it's dataset 😉